utils/__init__.py

```diff
@@ -72,8 +72,8 @@ def load_facebook_events(nodes, token=os.environ.get('FACEBOOK_TOKEN')):
                 continue
 
             # We need to reshape datetimes to events format
-            format_date = datetime.strptime(event['start_time'],
-                                            '%Y-%m-%dT%H:%M:%S%z')
```

Comment:
`%z` is Py3.2+, but the virtualenv is set to system python (2.7 is most linux systems). `%Z` is backward compatible.